composer.json

```diff
@@ -86,8 +86,9 @@
             },
             "drupal/url_embed": {
                 "WSOD if wrong url or network unavailable": "https://www.drupal.org/files/issues/url_embed_WSOD_convert_url_to_embed-2871744-5.patch",
-                "Improve performance for embeds": "https://www.drupal.org/files/issues/2018-03-16/url_embed-caching-layer-2867668-17.patch",
-                "Translate dialog title": "https://www.drupal.org/files/issues/2018-03-16/url_embed_translate_dialog_title-2953591-2.patch"
+                "Improve performance for embeds": "https://www.drupal.org/files/issues/2020-04-08/url_embed-n2867668-23.patch",
+                "Translate dialog title": "https://www.drupal.org/files/issues/2018-03-16/url_embed_translate_dialog_title-2953591-2.patch",
+                "Improve how the module deals with non-embeddable URLs": "https://www.drupal.org/files/issues/2020-08-17/urlembed-non-embeddable-urls-2761187-9-d8.patch"
```

Comment:
https://www.drupal.org/files/issues/2020-08-17/urlembed-non-embeddable-urls-2761187-9-d8.patch
@navneet0693 this isn't applying, probably due to their working branch being further ahead than beta1
we need a second patch which applies to beta1 for that

Reply:
![image](https://user-images.githubusercontent.com/8435994/90381551-2a276900-e09b-11ea-8b1e-8a4894a2495e.png)

![image](https://user-images.githubusercontent.com/8435994/90381563-2c89c300-e09b-11ea-94a2-10897f350cef.png)

It is not applying because of changes in https://www.drupal.org/project/url_embed/issues/2867668#comment-13544936

I am going re-roll it.

Reply:
I have added a custom here: https://www.drupal.org/project/social/issues/2930457 and it is now added in this PR.